spec/javascripts/health-care/health-care.spec.js

```diff
@@ -201,5 +211,31 @@ describe("Health Care Form", function() {
         expect(true).to.be.false;
       }).always(function() { done(); });
   });
+
+  it("should return true for valid monetary value input", function() {
+    var HealthApp = document.getElementById("fixture-frame").contentWindow.HealthApp;
+    var form = HealthApp.getFormRoot();
+    var testElement = form.elements['veteran[gross_wage_income]'];
+    var validInput = ['55000.00', '100', '1.99'];
+
+    for (var i = 0; i < validInput.length; i++) {
+      testElement.value = validInput[i];
+      var valid = HealthApp.validate(testElement);
+      expect(valid).to.be.true;
+    }
+  });
+
+  it("should return false for invalid monetary value input", function() {
+    var HealthApp = document.getElementById("fixture-frame").contentWindow.HealthApp;
+    var form = HealthApp.getFormRoot();
+    var testElement = form.elements['veteran[gross_wage_income]'];
+    var invalidInput = ['1,000', 'abc', '$100'];
+
+    for (var i = 0; i < invalidInput.length; i++) {
+      testElement.value = invalidInput[i];
+      var valid = HealthApp.validate(testElement);
+      expect(valid).to.be.false;
+    }
+  });
```

Comment:
No...verify that all the fields we care to collect have the data-validation-type="monetary" on them.

The test would look something like

```
expect($(form.elements('veteran[gross_wage_income]']).data('validation-type)).to.be.equal('monetary');
expect($(form.elements('veteran[net_business_income]']).data('validation-type)).to.be.equal('monetary');
```

The idea is, it's easy for a future maintainer to accidentally delete the attribute or typo something into the field. Therefore, we should assert it as an invariant to both document our intention and to ensure future maintainers can't do it accidentally.